MXLPersonalArmory/MXLPersonalArmory.csproj

```diff
@@ -102,9 +106,6 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Models\" />
-    <Folder Include="ViewModels\" />
-  </ItemGroup>
```

Comment:
Revert

Reply:
This change is required, because of the new folders and files.